test-end-to-end/android-maestro/common/accepting_course_join_request.yaml

```diff
@@ -8,10 +8,8 @@ appId: com.toughra.ustadmobile
 -  tapOn: "Courses"
 - tapOn: ${TESTCOURSENAME}
 - tapOn: "MEMBERS"
+- scroll
```

Comment:
This needs to be conditional - should use scrolluntil visible. Larger screen might not need scrolling (scroll might fail there)